src/index.html

```diff
@@ -11,7 +11,216 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+     id="form"
+     name="form"
+     action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+     method="post"
+    >
+      <fieldset
+       class="fieldset personal__information" 
+       name="personal__information">
+        <legend>
+          Personal information
+        </legend>
+      
+        <div class="block information">
+          <label for="username">
+            Surname:
+          </label>
+          <input 
+           id="username"
+           type="text"
+           name="username"
+           required
+           autocomplete="off"
+          >
```

Comment:
Pay attention on indents - it should ne 2 spaces. Fix it all over the page
```suggestion
          <input 
            id="username"
            type="text"
            name="username"
            required
            autocomplete="off"
          >
```